content/yaml-quick-start/relocating from AppCenter to Codemagic.md

```diff
@@ -0,0 +1,706 @@
+---
+title: Seamless migration from App Center to Codemagic
+description: How to ship your workflows to Codemagic
+weight: 1
+---
+
+## Quick comparison
+
+
+| **Features**                | **App Center** | **Codemagic**|
+|-----------------------------| ---------------|--------------|
+| `Swift/Objective-C iOS`     | `Yes`          | `Yes`        |
+| `Android apps`              | `Yes`          | `Yes`        |
+| `React Native builds`       | `Yes`          | `Yes`        |
+| `Flutter apps`              | `No`           | `Yes`        |
+| `Unity`                     | `Yes`          | `Yes`        |
+| `Ionic`                     | `Yes`          | `Yes`        |
+| `White-labeling solution`   | `No`           | `Yes`        |
+| `Automatic iOS code signing`| `No`           | `Yes`        |
+| `Manual iOS code signing`   | `Yes`          | `Yes`        |
+| `Android code signing`      | `Yes`          | `Yes`        |
+| `Automatic build versioning`| `Yes`          | `Yes`        |
+| `Running integrations tests`| `Yes`          | `Yes`        |
+| `Running unit tests`        | `Yes`          | `Yes`        |
+| `App Store publishing`      | `Yes`          | `Yes`        |
+| `Play Store publishing`     | `Yes`          | `Yes`        |
+| `Firebase App Distribution `| `No`           | `Yes`        |
+| `Running integrations tests`| `Yes`          | `Yes`        |
+| `Slack integration`         | `No`           | `Yes`        |
+| `Email notifications`       | `Yes`          | `Yes`        |
+| `macOS M2 support`          | `No`           | `Yes`        |
+| `Linux VM support`          | `No`           | `Yes`        |
+| `Windows VM support`        | `Yes`          | `Yes`        |
+| `Over-the-air-update`       | `Yes`          | `No`*        |
+| `Analytics`                 | `Yes`          | `No`*        |
+| `Apple device registration` | `No`           | `Yes`        |
+| `Remote access to VMs`      | `No`           | `Yes`        |
+| `Global environment variables`| `No`         | `Yes`        |
+| `Install apps from QR code` | `No`           | `Yes`        |
+
+
+* CodePush will continue standalone, so users can continue using the feature by App Center. Alternatively, Codemagic can be integrated with EAS updates.
```

Comment:
what is `EAS` ?

Reply:
Expo Application Services. they have `eas update` option which does what CodePush does. I will change it to its full name

Reply:
It is usually called EAS update